src/core/tests/opset.cpp

```diff
@@ -71,7 +71,7 @@ INSTANTIATE_TEST_SUITE_P(opset,
                                          OpsetTestParams{ov::get_opset10, 177},
                                          OpsetTestParams{ov::get_opset11, 177},
                                          OpsetTestParams{ov::get_opset12, 178},
-                                         OpsetTestParams{ov::get_opset13, 179}),
+                                         OpsetTestParams{ov::get_opset13, 180}),
```

Comment:
Why is 180, not 179?

Reply:
180 is the previous number +1, `BitwiseNot` op has been merged earlier